internal/docker/builder.go

```diff
@@ -457,7 +511,32 @@ func deployImage(
 	if err != nil {
 		return nil, err
 	}
+
 	containerID := body.ID
+
+	// Create the application service files
+	for asID, registration := range asIDToRegistrationMap {
+		var buf bytes.Buffer
+		tw := tar.NewWriter(&buf)
+		err = tw.WriteHeader(&tar.Header{
+			Name: fmt.Sprintf("/appservices/%s.yaml", asID),
```

Comment:
Please can we do filepath escapes here, as I would like to in the future allow untrusted Blueprint input.

Reply:
I assume you mean using `client.GjsonEscape`?

Reply:
I mean path escaping so you can't have an `asID` like `../../../etc/passwd`